golem-common/src/metrics.rs

```diff
@@ -405,3 +405,81 @@ pub mod api {
         };
     }
 }
+
+pub mod sqlite {
```

Comment:
i would consider to do it like generic 'db' metrics

with `type` label

so later we do not have to create set of root level metric for different DB/SQL like backend